src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionNavigationController.swift

```diff
@@ -82,19 +84,20 @@ class ExposureSubmissionNavigationController: UINavigationController, UINavigati
 	private(set) var button: ENAButton!
 	private(set) var secondaryButton: ENAButton!
 	private var bottomViewTopConstraint: NSLayoutConstraint!
-	private var exposureSubmissionService: ExposureSubmissionService?
+	private(set) var exposureSubmissionService: ExposureSubmissionService?
+	private weak var submissionDelegate: ExposureSubmissionNavigationControllerDelegate?
 
 	// MARK: - Initializers.
 
 	init?(
 		coder: NSCoder,
 		exposureSubmissionService: ExposureSubmissionService,
-		homeViewController: HomeViewController? = nil,
+		submissionDelegate: ExposureSubmissionNavigationControllerDelegate?,
```

Comment:
Removed the hard dependency to `HomeViewController`. `submissionDelegate` could now even come non-optional in `init` but this would break the tests and I did not want to make this PR bigger than needed. But this enabled this controller to be tested without a `HomeViewController`. This was possible already (by passing nil) but made it impossible to test certain things.